### PR TITLE
Use ssh instead of https for git origin

### PR DESCRIPTION
**Pull request by ixdy**
Typing usernames and passwords gets old fast.

### CODE REVIEW

**Comment by david-mcmahon** [comment]
Thanks for helping out with #65.  Is there any reason not to just make this the default and do away with https?

**Comment by ixdy** [comment]
Heh, I didn't even see #65. I was just trying to make things less painful for myself.

I don't know if everyone has set up ssh keys for GitHub. I guess if we defaulted to ssh this would fail pretty quickly if someone hasn't.

**Comment by ixdy** [comment]
OK, updated to just use ssh instead of https.

File: lib/gitlib.sh [hunk 1]
```diff
@@ -24,6 +24,7 @@ GHCURL="curl -s --fail --retry 10 -u ${GITHUB_TOKEN:-$FLAGS_github_token}:x-oaut
 JCURL="curl -g -s --fail --retry 10"
 K8S_GITHUB_API='https://api.github.com/repos/kubernetes/kubernetes'
 K8S_GITHUB_URL='https://github.com/kubernetes/kubernetes'
+K8S_GITHUB_SSH='git@github.com:kubernetes/kubernetes.git'
```
Comment (david-mcmahon) [inline lib/gitlib.sh]: Will this create kubernetes.git directories when the target is unspecified?  Maybe leave this without the .git for compatibility.

Comment (ixdy) [inline lib/gitlib.sh]: In both cases it checks out to `kubernetes/`. I can remove the `.git` if you want to be more consistent with the https URL, though the `.git` suffix is what GitHub supplies if you click the "Clone or Download" button on the repo's page.

Comment (ixdy) [inline lib/gitlib.sh]: yep, it all seems to work.

**Pull request merged**